src/Cli/dotnet/Commands/Test/TestModulesFilterHandler.cs

```diff
@@ -62,6 +63,6 @@ private static IEnumerable<string> GetMatchedModulePaths(string testModules, str
         Matcher matcher = new();
         matcher.AddIncludePatterns(testModulePatterns);
 
-        return MatcherExtensions.GetResultsInFullPath(matcher, rootDirectory);
+        return matcher.GetResultsInFullPath(rootDirectory);
```

Comment:
Weird that it was originally implemented that way

Reply:
My guess is someone did it that isn't familiar with extension methods. Extension methods is a strictly .NET language concept and I've never seen it in any other languages.